src/game.zig

```diff
@@ -894,6 +904,25 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 			Player.eyeVel[i] = firstTerm.mul(c_3.negate().subScalar(frictionCoefficient).mulScalar(0.5)).add(secondTerm.mul((c_3.subScalar(frictionCoefficient)).mulScalar(0.5))).val[0];
 			Player.eyePos[i] += firstTerm.add(secondTerm).addScalar(a/k).val[0];
 		}
+
+		// Calculate our FOV modifier based on our current speed.
+		{
+			const fovModStartSpeed: f32 = @as(f32, @floatCast(MaxSpeedWalking));
+			const fovModEndSpeed: f32 = @as(f32, @floatCast(MaxSpeedRunning));
+
+			// Clamped between 0-1. We'll let the renderer decide what the acual FOV values are
+			const fovModGoal : f32 =  std.math.clamp((@as(f32, @floatCast(movementSpeed)) - fovModStartSpeed) / (fovModEndSpeed - fovModStartSpeed), 0, 1);
```

Comment:
Have you though about using a logarithmic or logistic curve isntead of clamping to the max speed?
This could allow increases in fov beyond the maximum sprinting speed (like flying or hyperspeed)

Reply:
That's a good idea. I'll switch it out to something that dampens at higher values.